db/fixtures/chargeback_rates.yml

```diff
@@ -18,7 +18,7 @@
           :fixed_rate: 0.0
           :variable_rate: 0.02
     - :description: Used CPU Cores
-      :group: cpu
+      :group: cpu_container
```

Comment:
@simon3z, what we are changing here is the name of the group of metrics. This group includes metric that references cores of CPU in OpenShift projects ( correct me if I am wrong @Zeari). So,  we think that still that the name of the group would be cpu_container or cpu_openshift